disable GSS support if wrong Python library providing GSS bindings is installed
patch obtained from https://src.fedoraproject.org/rpms/python-paramiko/blob/master/f/paramiko-2.3.1-disable-gssapi-on-unsupported-version.patch
see also https://github.com/paramiko/paramiko/issues/1069
diff -ru paramiko-2.3.1.orig/paramiko/ssh_gss.py paramiko-2.3.1/paramiko/ssh_gss.py
--- paramiko-2.3.1.orig/paramiko/ssh_gss.py	2017-09-22 21:15:16.000000000 +0100
+++ paramiko-2.3.1/paramiko/ssh_gss.py	2017-10-29 21:16:08.071429184 +0100
@@ -51,7 +51,12 @@
 
 try:
     import gssapi
-    GSS_EXCEPTIONS = (gssapi.GSSException,)
+    try:
+        GSS_EXCEPTIONS = (gssapi.GSSException,)
+    except AttributeError:
+        # Unsupported GSS API
+        GSS_AUTH_AVAILABLE = False
+        _API = None
 except (ImportError, OSError):
     try:
         import pywintypes
